### PR TITLE
add transit extensions to python bindings

### PR DESCRIPTION
**Pull request by scullionw**
this changes the imports, so we need to update them if we already used bindings from this fork instead of the mobilitydata one (from a grep I don't think we do)

### CODE REVIEW

File: python/tests/test_gtfs_realtime_pb2.py [hunk 1]
```diff
@@ -16,15 +16,15 @@
...
+from google.transit.gtfs_spec.gtfs_realtime.proto import gtfs_realtime_pb2
 import unittest
 
 class GtfsRealtimeTest(unittest.TestCase):
 
   def testParseFromString(self):
     feed = gtfs_realtime_pb2.FeedMessage()
     data_dir = os.path.dirname(os.path.abspath(__file__))
-    with open(os.path.join(data_dir, 'vehicle_position.pb'), 'r') as f:
+    with open(os.path.join(data_dir, 'vehicle_position.pb'), 'rb') as f:
```
Comment (scullionw) [inline python/tests/test_gtfs_realtime_pb2.py:27]: even the test at mobilitydata repo is broken without this, weird

**Comment by juanborre** [comment]
How will we require this from the consumers? Should we add it to our Github registry?

**Comment by scullionw** [comment]
@juanborre for the analyzer I added it as a git dependency:

`gtfs-realtime-bindings-transit @ git+ssh://git@github.com/TransitApp/gtfs-realtime-bindings@pytransitexts#subdirectory=python`

since we don't have ci/cd for this repo I think its simpler?

**Comment by juanborre** [comment]
Okay 👍

**Pull request merged**